articles/azure-monitor/agents/azure-monitor-agent-troubleshoot-linux-vm-rsyslog.md

```diff
@@ -13,8 +13,8 @@ ms.reviewer: shseth
 Overview of Azure Monitor Agent for Linux Syslog collection and supported RFC standards:
```

Comment:
suggest to mention that as of version X it is TCP by default, previously domain socket